packages/cli/generators/controller/index.js

```diff
@@ -127,6 +127,16 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
             default: modelList[0],
             validate: utils.validateClassName,
           },
+          {
+            type: 'input',
+            name: 'modelNamePlural',
+            message:
+              'What is the custom plural form of the model (for building REST URL)',
+            when: this.artifactInfo.modelName === undefined,
+            default: response =>
+              utils.kebabCase(utils.pluralize(response.modelName)),
```

Comment:
Nice, +1 for using `kebabCase`.

Here is a wider context:

URLs like `/ProductReviews` are an anti-pattern in the REST world. The convention is to use [kebab case](http://wiki.c2.com/?KebabCase), e.g. `/product-reviews`.

In LB 3.x, we didn't want to break all existing applications, therefore we never changed the legacy behavior. There is a flag to enable kebab case though, see [rest.
normalizeHttpPath](https://loopback.io/doc/en/lb3/config.json.html#remoting-properties). 

I'd like us to leverage the opportunity to break backwards compatibility that LB 4.0 offers us, and fix the default REST APIs to finally use kebab-case instead of CamelCase in URL paths.

The impact on this pull request should be pretty minimal.

Reply:
Actually, since you added a dependency on [url-slug](https://www.npmjs.com/package/url-slug), I think we should use that one instead of `kebabCase`. What do you think?